tests/testsuite.src/backcomp_strings.at

```diff
@@ -0,0 +1,10029 @@
+AT_SETUP([INSPECT REPLACING LEADING ZEROS BY SPACES])
+AT_KEYWORDS([runmisc])
+
+AT_DATA([prog.c], [[
+/* Generated by           cobc 3.3-dev.0 */
```

Comment:
For all programs?

Reply:
For for all under "backcomp.at", yes.

Reply:
The test `TRANSFORM statement` from `tests/testsuite.src/run_misc.at` does not compile for GnuCOBOL 2.2 (but it works for 3.3~dev). What do you recommend (in case it also happens for other tests)?

I get a:
```
prog.cob: 19: error: syntax error, unexpected CHARACTERS, expecting FROM
```
error.

Reply:
There's no support for that in 2.2, so in this case testing backward-compatibility means testing 3.2 generated code.